### PR TITLE
Button conversion to pf4 component groupA

### PR DESCRIPTION
**Pull request by sg00dwin**

### CODE REVIEW

File: frontend/public/components/image-stream.tsx [hunk 1]
```diff
@@ -190,13 +190,10 @@ export const ExampleDockerCommandPopover: React.FC<ImageStreamManipulationHelpPr
         </div>
       }
     >
-      <button
-        className="btn btn-link btn-link--no-btn-default-values hidden-sm hidden-xs"
```
Comment (rhamilto) [inline frontend/public/components/image-stream.tsx]: Do you have a follow on task to remove the `btn-link--` classes?  Or are you hoping to get all instances in this PR and also remove them?

Comment (sg00dwin) [reply frontend/public/components/image-stream.tsx]: Yes, the plan is to remove any `.btn-link*` with a PR where all instances have been removed.

File: frontend/public/components/monitoring.tsx [hunk 2]
```diff
@@ -1261,14 +1261,14 @@ class SilenceForm_ extends React.Component<SilenceFormProps, SilenceFormState> {
                   </label>
                 </div>
                 <div className="col-xs-1">
-                  <button
+                  <Button
                     type="button"
-                    className="btn btn-link btn-link--inherit-color"
                     onClick={() => this.removeMatcher(i)}
                     aria-label="Remove matcher"
+                    variant="plain"
```
Comment (rhamilto) [inline frontend/public/components/monitoring.tsx]: This was a `link` before.  You intentionally changing it to `plain`?

Comment (sg00dwin) [reply frontend/public/components/monitoring.tsx]: This is just the `MinusCircleIcon` without text so those should be given plain instead of link

**Pull request merged**